packs/data/spells.db/unfolding-wind-blitz.json

```
@@ -0,0 +1,81 @@
+{
+    "_id": "Ww4cPZ3QHTSpaM1m",
+    "img": "systems/pf2e/icons/spells/wall-of-force.webp",
+    "name": "Unfolding Wind Blitz",
+    "system": {
+        "ability": {
+            "value": ""
+        },
+        "area": {
+            "areaType": "",
+            "value": ""
+        },
+        "category": {
+            "value": "focus"
+        },
+        "components": {
+            "focus": false,
+            "material": false,
+            "somatic": false,
+            "verbal": true
+        },
+        "cost": {
+            "value": ""
+        },
+        "damage": {
+            "value": {}
+        },
+        "description": {
+            "value": "<p>You blitz forward so quickly that you soar through the air, and nothing can react to you. @UUID[Compendium.pf2e.actionspf2e.Fly]{Fly} up to twice your Speed. This movement doesn't trigger reactions. At any point along the way, you can make Strikes with an unarmed attack against up to three different targets within your reach. You increase your multiple attack penalty for these attacks only after completing all of the attacks.</p>"
```

```suggestion
            "value": "<p>You blitz forward so quickly that you soar through the air, and nothing can react to you. Fly up to twice your Speed. This movement doesn't trigger reactions. At any point along the way, you can make Strikes with an unarmed attack against up to three different targets within your reach. You increase your multiple attack penalty for these attacks only after completing all of the attacks.</p>"
```
We don't normally link to basic action like stride, fly, or sustain.